core/CMakeLists.txt

```diff
@@ -31,6 +32,7 @@ target_link_libraries(gadgetron_core
 
 target_include_directories(gadgetron_core PUBLIC
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/toolboxes/mri_image>
```

Comment:
gadgetron_core is the deepest, most common part of Gadgetron. It doesn't depend on other bits of Gadgetron - **all** of Gadgetron depends on it.

I heartily dislike this dependency. 

Either part of the mri_image toolbox belongs in core (I doubt it), or the specializations belong in the mri_image toolbox.

Reply:
Fixed by refactoring the code.